packages/hl7v2-server/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "hl7v2-server",
+  "version": "1.0.0",
+  "description": "MLLP server that reads HL7 messages off of a raw tcp connection",
+  "main": "app.js",
+  "private": true,
+  "scripts": {
```

Comment:
Any particular reason to not follow the commands we have for the OSS api, like starting docker container, lint, etc.?

Reply:
Nope I copied the package.json from the term server i think, maybe I should have copied the one from the oss api

Reply:
addressed

Reply:
Not planning to add `start-docker-compose`?

```json
    "start-docker-compose": "docker compose -f docker-compose.dev.yml up --build -d && npm run start-opensearch",
```

Reply:
Will do this on a follow-up PR